### ISSUE TITLE
scripts not executable

### ISSUE THREAD

**Issue by nschloe** [TriBITSPub/TriBITS#125]
Debian reports warnings about [script-not-executable](https://lintian.debian.org/tags/script-not-executable.html) for
```
usr/lib/x86_64-linux-gnu/cmake/tribits/ctest_driver/makeCMakeCacheFile.sh
usr/lib/x86_64-linux-gnu/cmake/tribits/doc/build_docs.sh
usr/lib/x86_64-linux-gnu/cmake/tribits/doc/build_ref/create-build-ref.sh
usr/lib/x86_64-linux-gnu/cmake/tribits/doc/developers_guide/generate-dev-guide.sh
usr/lib/x86_64-linux-gnu/cmake/tribits/doc/developers_guide/scripts/dummy_test_commit.sh
usr/lib/x86_64-linux-gnu/cmake/tribits/doc/publish_docs.sh
```

**Comment by bartlettroscoe** [comment]
@nschloe,

This is actually a Trilinos defect, not a TriBITS defect, as it is Trilinos that is installing TriBITS.  See:

https://github.com/trilinos/Trilinos/blob/trilinos-release-12-6-branch/CMakeLists.txt#L98

The problem it seems is that, by default, the CMake `INSTALL()` command does **not** maintain source permissions.  It seems that you have to pass in `USE_SOURCE_PERMISSIONS`.  See:

http://stackoverflow.com/questions/13615995/how-to-preserve-file-permissions-with-cmake-install-directory-directive

If I patch the Trilinos 12.6 branch, and if Brent puts out another patch release of Trilinos 12.6 (i.e. 12.6.4), then would you update the Debian Trilinos 12.6 distribution for this?  If not, then I might as well just do #114 to add support for TriBITS installs in TriBITS itself and then snapshot TriBITS into the Trilinos 'master' (soon to be 'develop') branch.  Then this would be in the next major release of Trilinos (i.e. 12.8).

**Comment by nschloe** [comment]
Those are "just" warnings, so it'd be okay to have them fixed in 12.8 too.

**Comment by bartlettroscoe** [comment]
> Those are "just" warnings, so it'd be okay to have them fixed in 12.8 too.

That is good.  

To address this, I have added `USE_SOURCE_PERMISSIONS` to the scope of #114 and when it is done, I will snapshot TriBITS into Trilinos and this issue should be resolved.

I will therefore close this issue ticket and just raise the priority of #114 to make sure that it gets done before the next Trilinos release.
